ethcore/src/client/config.rs

```diff
@@ -16,15 +16,15 @@
 
 use std::str::FromStr;
 
-use verification::{VerifierType, QueueConfig};
+use blockchain::Config as BlockChainConfig;
 use journaldb;
 use snapshot::SnapshotConfiguration;
+use trace::Config as TraceConfig;
 use types::client_types::Mode;
+use verification::QueueConfig;
```

Comment:
```suggestion
use verification::{VerifierType, QueueConfig};
```
(and remove line 27)